documentation/user-interface/test-editor.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: docs
+title: Test Editor
+permalink: /documentation/user-interface/test-editor.html
+---
+# Test Editor
+
+The **Test** editor allows you to view and modify existing tests, and create new ones. To open a test in the editor, double-click it in the [Explorer](explorer-panel.md) panel or click it in the [Fixture](fixture-editor.md) editor. When you [create an empty test](../working-with-testcafe-studio/recording-tests/README.md#creating-tests) or start [recording a new test](../working-with-testcafe-studio/recording-tests/README.md#starting-and-stopping-recording), TestCafe Studio automatically opens the test in the editor.
+
+The **Test** editor consists of the following parts:
+
+* The test area - Displays the current test's content.
+* The **Actions** panel - Lists [actions](../working-with-testcafe-studio/recording-tests/test-actions/README.md) you can add to the test.
+    The actions are organized into categories. You can expand or collapse the categories by clicking the arrows next to the categories' names.
+* The toolbar - Provides common information about the test (test name, fixture name, tested page's URL) and buttons for perfoming common tasks with it.
```

Comment:
I think we also should say (for fixture and test) that:
 - fixture name in test view is clickable and leads to corresponding fixture view
- page url is clickable fot them both and open page in default browser